splunk_handler/__init__.py

```diff
@@ -137,8 +125,31 @@ def emit(self, record):
             # Flush log immediately; is blocking call
             self._splunk_worker(payload=record)
 
+    def close(self):
+        self.shutdown()
+        logging.Handler.close(self)
+
+    #
+    # helper methods
+    #
+
+    def start_worker_thread(self):
+        # Start a worker thread responsible for sending logs
+        if self.flush_interval > 0:
+            self.write_debug_log("Preparing to spin off first worker thread Timer")
+            self.timer = Timer(self.flush_interval, self._splunk_worker)
+            self.timer.daemon = True  # Auto-kill thread if main process exits
+            self.timer.start()
+
+    def write_log(self, log_message):
+        print("[SplunkHandler] " + log_message)
+
+    def write_debug_log(self, log_message):
+        if self.debug:
+            print("[SplunkHandler DEBUG]" + log_message)
```

Comment:
Lost the whitespace after the closing `]`.  Should be `print("[SplunkHandler DEBUG] " + log_message)`.